packages/calypso-color-schemes/src/shared/color-schemes/_sunrise.scss

```diff
@@ -0,0 +1,140 @@
+/*
+Sunrise
+
+This is a theme ported directly from wp-admin as a part of the nav unification
+project.
+
+WP Admin Unthemed definitions:
+
+$text-color: #fff !default;
+$base-color: #23282d !default;
+$icon-color: hsl( hue( $base-color ), 7%, 95% ) !default;
+$highlight-color: #0073aa !default;
+$notification-color: #d54e21 !default;
+
+WP Admin Sunrise Definition:
+
+$base-color: #cf4944;
+$icon-color: None, so ... using the wp-admin definition above: = #f3f1f1
+$highlight-color: #dd823b;
+$notification-color: #ccaf0b;
+$menu-submenu-focus-text: lighten( $highlight-color, 35% ) = #f7e3d3
+
+Well use studio-orange for both the primary and accent colors
+*/
+
+// Set up variables for Calypso Sunrise
+:root {
+    --theme-text-color: #ffffff; /* Direct from wp-admin */
+    --theme-text-color-rgb: 255, 255, 255; /* Manual conversion */
+    --theme-base-color: #cf4944; /* Direct from wp-admin */
+    --theme-base-color-rgb: 207, 73, 68; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-submenu-background-color: #be3631; /* darken( $base-color, 7% ), computed: http://scg.ar-ch.org/
+    --theme-icon-color: #f3f1f1; /* Direct from wp-admin */
+    --theme-highlight-color: #dd823b; /* Direct from wp-admin */
+    --theme-highlight-color-rgb: 221, 130, 59; /* Manually computed https://www.rapidtables.com/convert/color/hex-to-rgb.html */
+    --theme-notification-color: #ccaf0b; /* Direct from wp-admin */
+
+}
```

Comment:
As also suggested in https://github.com/Automattic/wp-calypso/pull/47122 let's try consolidating these variables in _defaut.scss.

Reply:
Done